nlbq/config.py

```diff
@@ -0,0 +1,20 @@
+from functools import lru_cache
+
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    openai_api_key: str = None
+    google_application_credentials: str = None
+    uvicorn_host: str = "0.0.0.0"
+    uvicorn_port: int = 8080
+    prompt_template_file: str = "prompt.txt"
+
+    class Config:
+        env_file = ".env"
+        env_file_encoding = "utf-8"
+
+
+@lru_cache
```

Comment:
Wouldn't this only be called at app startup anyway?

Reply:
[https://fastapi.tiangolo.com/advanced/settings/ recommends this pattern](https://fastapi.tiangolo.com/advanced/settings/#creating-the-settings-only-once-with-lru_cache) explains this

given we load it in several places I thought it would be useful to follow